opensuse-leap-ports-repoindex.xml

```diff
@@ -23,4 +23,10 @@
     enabled="false"
     autorefresh="true"/>
 
+<repo url="https://codecs.opensuse.org/openh264/openSUSE_Leap"
```

Comment:
IIRC the cisco server does not have valid https:// setup, so we cannot use https here and then redirect to HTTP or redirect to a non-valid https:// setup